<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>readsHistogramWidget</class>
 <widget class="QWidget" name="readsHistogramWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>930</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>930</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Features Reads Distribution</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1274</width>
        <height>904</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QWidget" name="plotNormalWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>1200</width>
             <height>800</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1200</width>
             <height>800</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Save the DEA to a file</string>
           </property>
           <property name="statusTip">
            <string>Save the DEA to a file</string>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
           <property name="icon">
            <iconset resource="../../../st_client-build/application.qrc">
             <normaloff>:/images/filesave.png</normaloff>:/images/filesave.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Close the DEA</string>
           </property>
           <property name="statusTip">
            <string>Close the DEA</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="icon">
            <iconset resource="../../../st_client-build/application.qrc">
             <normaloff>:/images/abort.png</normaloff>:/images/abort.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../st_client-build/application.qrc"/>
 </resources>
 <connections/>
</ui>
